apps/cricketworldcup/cricket_world_cup.star

```diff
@@ -0,0 +1,700 @@
+                pad = (2, 1, 0, 0),
+                child = render.Marquee(
+                    width = 50,
+                    child = render.Text(content = Batsman, color = BatsmanColor, font = "CG-pixel-3x5-mono", offset = 0),
+                ),
+            )),
+            render.Box(width = 14, height = 8, child = render.Padding(
+                pad = (0, 0, 0, 0),
+                child = render.Marquee(
+                    width = 14,
+                    child = render.Text(content = Runs, color = BatsmanColor, font = "CG-pixel-3x5-mono", offset = 0),
+                ),
+            )),
+        ],
+    )
+
+def StatusRow(StatusMsg, StatusColor):
+    return render.Row(
+        children = [
+            render.Box(
+                width = 64,
+                height = 8,
+                child = render.Text(content = StatusMsg, color = StatusColor, font = "CG-pixel-3x5-mono"),
+            ),
+        ],
+    )
+
+TeamOptions = [
+    schema.Option(
+        display = "England",
+        value = "1",
+    ),
+    schema.Option(
+        display = "Australia",
+        value = "2",
+    ),
+    schema.Option(
+        display = "South Africa",
+        value = "3",
+    ),
+    schema.Option(
+        display = "New Zealand",
+        value = "5",
+    ),
+    schema.Option(
+        display = "India",
+        value = "6",
+    ),
+    schema.Option(
+        display = "Pakistan",
+        value = "7",
+    ),
+    schema.Option(
+        display = "Sri Lanka",
+        value = "8",
+    ),
+    schema.Option(
+        display = "Afghanistan",
+        value = "40",
+    ),
+    schema.Option(
+        display = "Bangladesh",
+        value = "25",
+    ),
+    schema.Option(
+        display = "Netherlands",
+        value = "15",
+    ),
+]
+
+def getTeamColor(teamID):
+    if teamID == 2:  # Australia
+        return ("#ffdd00")
+    elif teamID == 3:  # South Africa
+        return ("#04822B")
+    elif teamID == 7:  # Pakistan
+        return ("#04822B")
+    elif teamID == 5:  # New Zealand
+        return ("#fff")
+    elif teamID == 1:  # England
+        return ("#2B57AC")
+    elif teamID == 6:  # India
+        return ("#137dd2")
+    elif teamID == 8:  # Sri Lanka
+        return ("#203d89")
+    elif teamID == 25:  # Bangladesh
+        return ("#006A4E")
+    elif teamID == 15:  # Netherlands
+        return ("#F36C21")
+    elif teamID == 40:  # Afghanistan
+        return ("#fff")
+    else:  # For any other team
+        return ("#fff")
+
+def get_schema():
+    return schema.Schema(
+        version = "1",
+        fields = [
+            schema.Dropdown(
+                id = "TeamList",
+                name = "Team",
+                desc = "Choose your team",
+                icon = "gear",
+                default = TeamOptions[0].value,
+                options = TeamOptions,
+            ),
+        ],
+    )
+
+def get_cachable_data(url, timeout):
```

Comment:
Would be great if this could be updated to rely on the http modules built-in cache functionality. Just pass `ttl_seconds` to `http.get()` and you'll be able to remove the dependency on `cache.star`.